easybuild/easyconfigs/q/QuantumESPRESSO/QuantumESPRESSO-7.4-foss-2024a.eb

```diff
@@ -0,0 +1,157 @@
+            "repo_name": "libmbd",
+            "commit": local_mbd_hash,
+            'clone_into': 'mbd',
+        },
+    },
+    {
+        "filename": "devxlib-%s.tar.gz" % local_devxlib_hash,
+        "git_config": {
+            "url": "https://gitlab.com/max-centre/components",
+            "repo_name": "devicexlib",
+            "commit": local_devxlib_hash,
+            'clone_into': 'devxlib',
+        },
+    },
+    {
+        "filename": "d3q-%s.tar.gz" % local_d3q_hash,
+        "git_config": {
+            "url": "https://github.com/anharmonic",
+            "repo_name": "d3q",
+            "commit": local_d3q_hash,
+        },
+    },
+    {
+        "filename": "fox-%s.tar.gz" % local_fox_hash,
+        "git_config": {
+            "url": "https://github.com/pietrodelugas",
+            "repo_name": "fox",
+            "commit": local_fox_hash,
+        },
+    },
+    {
+        "filename": "qe-gipaw-%s.tar.gz" % local_qe_gipaw_hash,
+        "git_config": {
+            "url": "https://github.com/dceresoli",
+            "repo_name": "qe-gipaw",
+            "commit": local_qe_gipaw_hash,
+        },
+    },
+    {
+        "filename": "pw2qmcpack-%s.tar.gz" % local_qmcpack_hash,
+        "git_config": {
+            "url": "https://github.com/QMCPACK",
+            "repo_name": "pw2qmcpack",
+            "commit": local_qmcpack_hash,
+        },
+    },
+    {
+        "filename": "wannier90-%s.tar.gz" % local_w90_hash,
+        "git_config": {
+            "url": "https://github.com/wannier-developers",
+            "repo_name": "wannier90",
+            "commit": local_w90_hash,
+        },
+    },
+]
+patches = [
+    {
+        'name': 'QuantumESPRESSO-7.4-d3q.patch',
+        'sourcepath': '../'  # Needed as patches are normally applied to the first `finalpath` directory
+    },
+]
+# Holding off checksum checks untill 5.0.x
+# https://github.com/easybuilders/easybuild-framework/pull/4248
+# checksums = [
+#     {'q-e-qe-7.4.tar.gz': '2c58b8fadfe4177de5a8b69eba447db5e623420b070dea6fd26c1533b081d844'},
+#     {'lapack-%s.tar.gz' % local_lapack_hash: 'c05532ae0e5fe35f473206dda12970da5f2e2214620487d71837ddcf0ea6b21d'},
+#     {'mbd-%s.tar.gz' % local_mbd_hash: 'a180682c00bb890c9b1e26a98addbd68e32f970c06439acf7582415f4c589800'},
+#     {'devxlib-%s.tar.gz' % local_devxlib_hash: '76da8fe5a2050f58efdc92fa8831efec25c19190df7f4e5e39c173a5fbae83b4'},
+#     {'d3q-%s.tar.gz' % local_d3q_hash: '43e50753a56af05d181b859d3e29d842fb3fc4352f00cb7fe229a435a1f20c31'},
+#     {'fox-%s.tar.gz' % local_fox_hash: '99b6a899a3f947d7763aa318e86f9f08db684568bfdcd293f3318bee9d7f1948'},
+#     {'qe-gipaw-%s.tar.gz' % local_qe_gipaw_hash: '9ac8314363d29cc2f1ce85abd8f26c1a3ae311d54f6e6034d656442dd101c928'},
+#     {'pw2qmcpack-%s.tar.gz' % local_qmcpack_hash: 'a8136da8429fc49ab560ef7356cd6f0a2714dfbb137baff7961f46dfe32061eb'},
+#     {'wannier90-%s.tar.gz' % local_w90_hash: 'f989497790ec9777bdc159945bbf42156edb7268011f972874dec67dd4f58658'},
```

Comment:
please update the checksums?

Reply:
I see [framework:db2ab3ac3ac5a654b85a7c59eae0ecd6c9f66e07](https://github.com/easybuilders/easybuild-framework/commit/db2ab3ac3ac5a654b85a7c59eae0ecd6c9f66e07)  is still not an ancestor of `develop`, only `5.0.x`.
Should i proceed regardless?

Reply:
ok, then delete them because they are wrong, but q-e-qe-7.4.tar.gz can be downloaded directly, so that one should be updated

Reply:
Fixed with [0d566e2](https://github.com/easybuilders/easybuild-easyconfigs/pull/21866/commits/0d566e293293a1cdca989a0f47b74de8a90df6fc)